CODEOWNERS

```
@@ -16,6 +16,7 @@ tap_migrations.json               @Homebrew/tsc @MikeMcQuaid
 LICENSE.txt                       @Homebrew/tsc @MikeMcQuaid
 .github/workflows/                @Homebrew/core @MikeMcQuaid 
 
+audit_exceptions/linux_only_gcc_dependency_allowlist.json               @Homebrew/tsc @carlocab
```

```suggestion
audit_exceptions/linux_only_gcc_dependency_allowlist.json               @Homebrew/tsc @Bo98
```
@Bo98 you ok if we do this instead?

---

```suggestion
audit_exceptions/linux_only_gcc_dependency_allowlist.json               @Homebrew/tsc
```

I think it's fine for it to be just the TSC here.